lxd/main_activateifneeded.go

```diff
@@ -116,36 +129,45 @@ func (c *cmdActivateifneeded) run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	instances, err := instance.LoadNodeAll(d.State(), instancetype.Any)
+	// Don't call d.State() as that will run instance driver feature checks (amongst other unnecessary things).
+	// We just need access to the DBs.
+	s := &state.State{
+		DB:          d.db,
+		LocalConfig: d.localConfig,
+	}
+
+	// Because we've not loaded the cluster member name from the global DB, this function will load all
+	// instances in the database and not filter by local member name. However as we only do this when
+	// core.https_address is unset (as there is an early check above) this is not inefficient because we will
+	// only get to this part for standalone servers.
+	instances, err := instance.LoadNodeAll(s, instancetype.Any)
 	if err != nil {
 		return err
 	}
 
 	for _, inst := range instances {
 		if instanceShouldAutoStart(inst) {
 			logger.Debugf("Daemon has auto-started instances, activating...")
```

Comment:
It's too bad we don't have a linter flagging those `logger.XXXf()` calls lacking format specifier cause we should be using `logger.XXX()`.

Reply:
Cool thanks

Reply:
https://github.com/canonical/lxd/pull/15924